app/views/svie/pdf/_inactive_member.html.erb

```diff
@@ -1,6 +1,5 @@
-<h1 class="text-center">Tagfelvételi kérelem <br> öreg tagsághoz</h1>
 <p>
-  <%= render "svie/pdf/personal_data", user: user %>
+  <%= render "svie/pdf/personal_data", user: @user %>
```

Comment:
```suggestion
  <%= render 'personal_data', user: @user %>
```
Here, if I'm right, you can leave the full path as the default partial lookup starts in the current folder.